packages/blade/src/components/TopNav/docs/TopNav.stories.tsx

```diff
@@ -42,12 +44,59 @@ import { Text } from '~components/Typography';
 import { Menu, MenuFooter, MenuHeader, MenuItem, MenuOverlay } from '~components/Menu';
 import { Link as BladeLink } from '~components/Link';
 import { Badge } from '~components/Badge';
+import { getStyledPropsArgTypes } from '~components/Box/BaseBox/storybookArgTypes';
+import { Sandbox } from '~utils/storybook/Sandbox';
+import StoryPageWrapper from '~utils/storybook/StoryPageWrapper';
+
+import { Alert } from '~components/Alert';
+import { List, ListItem } from '~components/List';
+import { makeSize } from '~utils';
+import {
+  SIDE_NAV_EXPANDED_L1_WIDTH_DESKTOP,
+  SIDE_NAV_EXPANDED_L1_WIDTH_MOBILE,
+} from '~components/SideNav/tokens';
+
+const DocsPage = (): React.ReactElement => {
+  return (
+    <StoryPageWrapper
+      componentName="TopNav"
+      componentDescription="The top navigation bar is positioned at the top of the screen that provides quick access to different products, search & user profile."
+      figmaURL=""
+    >
+      <Title>Usage (with React Router v6)</Title>
+      <Alert
+        color="notice"
+        title="State Management Note"
+        description="TopNav component requires you to handle active link and active menu item on consumer end
+        since the component is detached from React Router. The example below includes some boilerplate in handling these active states using React Router v6. Make sure to test your edge cases while implementing."
+        isFullWidth
+        isDismissible={false}
+      />
+
+      <Sandbox
+        files={topNavFullExample}
+        editorHeight={600}
+        hideNavigation={false}
+        openFile="SideNavExample.tsx,utils.tsx,App.tsx,TopNavExample.tsx"
+      />
+    </StoryPageWrapper>
+  );
+};
 
 export default {
   title: 'Components/TopNav',
   component: TopNav,
+  tags: ['autodocs'],
+  argTypes: {
+    ...getStyledPropsArgTypes({ category: 'StyledProps' }),
```

Comment:
This will add all the styled-props but we only support few no?

Reply:
Yeah, removed, I think the autodocs will just work since we only support few styled props